pom.xml

```diff
@@ -68,6 +68,11 @@
       <version>1.23.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
It looks like some of the commits from your previous PR (#31) are showing up in this one as well, do you need us to approve that PR before you merge this one?

Reply:
Oh yes, good catch!